templates/.snapshots/TestDatadogTf-monitoring-datadog.tf.tpl-monitoring-datadog.tf.snapshot

```diff
@@ -38,6 +38,32 @@ locals {
   ddTags = concat(["testing", "team:test:team"], var.additional_dd_tags)
 }
 
+resource "datadog_monitor" "argocd_application_health_status" {
+  type = "query alert"
+  name = "Testing argocd application health status"
+  query = "max(last_15m):clamp_max(sum:argocd_application_controller.argocd_app_info{name:testing,health_status:healthy} by {cluster_name}, 1) < 1"
```

Comment:
Does this handle missing data? It kind of looks like you'll end up with a "No data" warning if this app doesn't exist. You probably need `default_zero` ... with `fill`? The docs seem to imply you can combine the two for a nonzero fill. :)